classes/import-export.php

```diff
@@ -514,6 +515,24 @@ public function import_csv() {
 		}
 
 		/**
+		 * Gets import path.
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->